src/quo/components/selectors/disclaimer/style.cljs

```diff
@@ -3,16 +3,16 @@
     [quo.foundations.colors :as colors]))
 
 (defn container
-  [blur?]
+  [blur? theme]
```

Comment:
Maybe we can pass a map here?

Reply:
I'm fine with two arguments 👍
maybe it more around 3 or more args when a map can be used?